modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talo-platform-prepare-end-to-end_{context}"]
+= End-to-end procedures for updating clusters in a disconnected environment
+
+If you have deployed spoke clusters with distributed unit (DU) profiles using the GitOps ZTP with Topology Aware Lifecycle Operator pipeline described in "Deploying distributed units at scale in a disconnected environment", this procedure tells you how to upgrade your spoke clusters and Operators.
+
+[id="talo-platform-prepare-for-update_{context}"]
+== Preparing for the updates
+
+If both the hub and the spoke clusters are running {product-title} 4.9, you need to update ZTP from version 4.9 to 4.10. If {product-title} 4.10 is used, you can set up the environment.
+
+[id="talo-platform-prepare-for-update-env-setup_{context}"]
+== Setting up the environment
+
+If you deployed ZTP in a disconnected environment as instructed in the "Preparing the disconnected environment" section in "Deploying distributed units at scale in a disconnected environment", you need to do the following steps before TALO can successfully perform update operations:
+
+TALO can perform both platform and Operator updates:
+
+* For platform updates, you have to do the following:
++
+. Mirror the desired {product-title} image repository. For more information about how to mirror an image repository, see the "Mirroring the OpenShift Container Platform image repository" section.
++
+[IMPORTANT]
+====
+The `imageContentSources` section, which can be retrieved with the `oc adm release mirror` command, needs to be saved for platform updates.
+For example, this is the output of the `imageContentSources` which contains the information of your mirrors:
```

Comment:
We shouldn't be using commands in admonition blocks as they have the risk of getting missed in the sequence. Can this be restructured as a step?
```suggestion
The `imageContentSources` section, which can be retrieved with the `oc adm release mirror` command, must be saved for platform updates.
For example, this is the output of the `imageContentSources` which contains the information of your mirrors:
```